docs/custom_installation.md

```diff
@@ -0,0 +1,56 @@
+## Custom Installation
+--------------------
+
+__If you are installing this module using a control-repo, you must have splunk_hec in your production environment's Puppetfile so the puppetserver process can load the libraries it needs properly. You can then create a feature branch to enable them and test the configuration, but the libraries must be in production otherwise the feature branch wont work as expected.__
```

Comment:
I believe so, but I'll check with @mrzarquon .

Reply:
Yes, this does apply to whatever environment that the master is in.

Reply:
Something like the following may help to make it clear for that use case.

```suggestion
__If you are installing this module using a control-repo, you must have splunk_hec in your production environment's Puppetfile so the puppetserver process can load the libraries it needs properly. You can then create a feature branch to enable them and test the configuration, but the libraries must be in production otherwise the feature branch won't work as expected. If your puppetserver is in a different environment, please add this module to the Puppetfile in that environment as well__
```